README.md

```diff
@@ -177,6 +177,31 @@ Call a gRPC endpoint using a JSON input. What this does behind the scenes:
 
 *See [docs/grpc.md](docs/grpc.md) for full instructions.*
 
+### SSL/TLS Connections
+
+To enable SSL/TLS connections to the server, use the `--tls` command line flag. 
+If not specified all the following flags will be ignored.
+
+By default host validation is enabled. To disable, pass the `--insecure` command line flag.
+
+To use a custom certificate authority, pass the `--cacert` command line flag to the path of the file in pem format.
+
+To use a custom key and certificate, pass the `--key` and `--cert` command line flag to the path of the files in pem format.
+
+To use a custom server name for validation, pass `--server-name` command line flag.
```

Comment:
What happens if I don't use this flag?

All flags need their default behavior explained. https://godoc.org/github.com/fullstorydev/grpcurl#ClientTransportCredentials has some explanation on the defaults, but we need a full overview here.

Reply:
I'll update the readme when I move it to the suggested location and incorporate this comment.